Update README file (#81)
googleinterns/step6-2020: README.md

# Support Local Businesses

### Objective

This is our Google 2020 [STEP internship](https://buildyourfuture.withgoogle.com/programs/step/) project. 

> Support Local Businesses is a platform where businesses can share how the COVID-19 Pandemic affected 

Reviewer: Change to Bloom

Author: done

them and reach out to the general public for assistance virtually. 

> We build for the local and global small businesses around the world that are being affected 

Reviewer: you can remove "local and global"
you don't really need local since that makes it sound like we are targeting a smaller region.
you don't need global since you say "around the world" after this

Author: done!

> from the COVID-19 Pandemic. We know that this pandemic has affected numerous small businesses 

Reviewer 2: ```suggestion
by the COVID-19 Pandemic. We know that this pandemic has affected numerous small businesses 
```

Author: done

> and we want to create a platform to support and bridge the community together during this time.

Reviewer 2: ```suggestion
and we want to create a platform to support and unite communities during this difficult time.
```

Author: done

> Link to our website: https://cloud-build-step-2020.uc.r.appspot.com/

Reviewer 2: ```suggestion
Link to our website: https://cloud-build-step-2020.appspot.com/
```

Author: done
